core/src/avm2/globals/flash/events/NativeWindowDisplayStateEvent.as

```diff
@@ -0,0 +1,49 @@
+// The initial version of this file was autogenerated from the official AS3 reference at
+// https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/events/NativeWindowDisplayStateEvent.html
+// by https://github.com/golfinq/ActionScript_Event_Builder
+// It won't be regenerated in the future, so feel free to edit and/or fix
+
+package flash.events
+{
+  [API("661")]
+  public class NativeWindowDisplayStateEvent extends Event
+  {
+    public static const DISPLAY_STATE_CHANGE:String = "displayStateChange";
```

Comment:
There are two constants documented, only 1 here:
 - `DISPLAY_STATE_CHANGING`
- `DISPLAY_STATE_CHANGE`

The event builder script can probably only handle one constant per event.